examples/BRD_test.py

```diff
@@ -0,0 +1,97 @@
+# for 1D BRD, adapted mainly from Venkataramanan 2002
+# but checked against BRD 1981
+from pyspecdata import *
+from scipy.optimize import nnls
+init_logging('debug')
+vd_list = nddata(linspace(5e-4,10,25),'vd')
+t1_name = r'$\log(T_1)$'
+logT1 = nddata(r_[-4:2:100j],t1_name)
+def Gaussian_1d(axis,mu1,sigma1):
+    this_G = exp(-(axis-mu1)**2/2/sigma1**2)
+    return this_G
+true_F = Gaussian_1d(logT1.C.run(lambda x: 10**(x)),6,0.3)
```

Comment:
OK -- so actually this looks like an example -- does the file in "demos" do anything this does not?

Reply:
I believe the script in "demos" was meant to just test the threading capabilities when that had been implemented, also to assess whether it was saving us time or not

Reply:
ahh -- that makes sense -- if we wanted to make an example to test the threading capabilities, we could do that.